tests/test-xdp-utils.c

```diff
@@ -146,11 +146,6 @@ test_app_id_via_systemd_unit (void)
   g_assert_cmpstr (app_id, ==, "org.gnome.Epiphany");
   g_clear_pointer (&app_id, g_free);
 
-  app_id = _xdp_parse_app_id_from_unit_name ("app-glib-spice\\x2dvdagent-1839.scope");
-  /* App IDs must have two periods */
-  g_assert_cmpstr (app_id, ==, "");
-  g_clear_pointer (&app_id, g_free);
-
```

Comment:
I'd keep this test-case, and change the expected result to

```
  g_assert_cmpstr (app_id, ==, "spice-vdagent");
```

which would make it very obvious that this commit is a behaviour change.